README.md

```diff
@@ -204,6 +204,40 @@ Attributes:
 <img src="https://raw.githubusercontent.com/nautobot/nautobot-plugin-circuit-maintenance/develop/docs/images/circuit_maintenance.png" width="800" class="center">
 </p>
 
+### Metrics
+
+Leveraging on `nautobot-capacity-metrics` plugin, the `nautobot-circuit-maintenance` plugin will expose application metrics at `/api/plugins/capacity-metrics/app-metrics`
+
+Current exposed metric is the `circuit operational status` which shows the operational status for each `Circuit`(attached to a `CircuitTermination`) depending on related Circuit Maintenances (1: Operational, 2: Under active maintenance):
+
+```
+# HELP nautobot_circuit_maintenance_total Circuit operational status
+# TYPE nautobot_circuit_maintenance_total counter
+nautobot_circuit_maintenance_total{circuit="1111111",circuit_type="Transit",provider="ntt",site="Barcelona"} 2.0
+nautobot_circuit_maintenance_total{circuit="2222222",circuit_type="Peering",provider="colt",site="Girona"} 1.0
+```
```

Comment:
Ignorant question - is this a common/useful pattern in Prometheus? I naively would have expected:

1) this to be a Gauge, not a Counter
2) The status to be a distinct label with a value for each possible state (with gauge value 0 or 1 accordingly)

I.e. something like:

```
# HELP circuit_maintenance_status Circuit operational status
# TYPE circuit_maintenance_status gauge
circuit_maintenance_status{circuit="1111111",circuit_type="Transit",provider="ntt",site="Barcelona",status="operational"} 0.0
circuit_maintenance_status{circuit="1111111",circuit_type="Transit",provider="ntt",site="Barcelona",status="active-maintenance"} 1.0
circuit_maintenance_status{circuit="2222222",circuit_type="Peering",provider="colt",site="Girona",status="operational"} 1.0
circuit_maintenance_status{circuit="2222222",circuit_type="Peering",provider="colt",site="Girona",status="active-maintenance"} 0.0
```

Reply:
yes, I also believe that gauge makes more sense as it is not incremental

Reply:
Gauge make sense but I would prefer to keep the value as 1 & 2 if possible, it's more inline what the industry is doing